sdk/eventhub/eventhubs-checkpointstore-table/src/tableCheckpointStore.ts

```diff
@@ -62,15 +212,27 @@ export class TableCheckpointStore implements CheckpointStore {
    *  - `abortSignal`: A signal used to request operation cancellation.
    *  - `tracingOptions`: Options for configuring tracing.
    */
-  async listCheckpoints(): /*
+  async listCheckpoints(
     fullyQualifiedNamespace: string,
     eventHubName: string,
-    consumerGroup: string,
-    options: OperationOptions = {}
-    */
-  Promise<Checkpoint[]> {
-    console.log("no checkpoints to list");
-    throw new Error("not implemented");
+    consumerGroup: string
+  ): Promise<Checkpoint[]> {
+    const partitionKey = `${fullyQualifiedNamespace} ${eventHubName} ${consumerGroup} Checkpoint`;
+    const checkpoints: Checkpoint[] = [];
+    const entitiesIter = this._tableClient.listEntities<CheckpointEntity>({
+      queryOptions: { filter: odata`PartitionKey eq ${partitionKey}` }
+    });
+    for await (const entity of entitiesIter) {
+      checkpoints.push({
+        consumerGroup,
+        eventHubName,
+        fullyQualifiedNamespace,
+        partitionId: entity.rowKey,
+        offset: entity.offset,
```

Comment:
It's possible other languages will save offset/sequencenumber as a string (looks like python currently saves offset as a string) so to be safe, I think it's worth wrapping those fields in a parseInt call.